free-2.md

```diff
@@ -1,44 +1,85 @@
 # Free project 2
 
+MarkTeX?? : Simple and possible to use in a WYSIWYG editor like Markdown,
+but targeted at LaTeX's domain.
```

Comment:
Markdown is a DSL but I'm not sure how it's an editor or how it's WYSIWYG. I thought it was a lot like LaTeX in that it is converted to HTML, although with a strong emphasis on easy-to-read source